index.bs

```diff
@@ -651,3 +779,81 @@ The [=remote end steps=] are:
 
 2. Return [=success=] with data |body|
 
+### Subscribe ### {#command-session-subscribe}
+
+The <dfn export for=commands>subscribe</dfn> command enables certain events either
+globally or for a set of browsing contexts
```

Comment:
Having read through **update the event map** I'm not sure that enabling globally really works. Is the intended model to maintain a global event map, or to enable/disable on all targets which currently exist? (A similar question asked elsewhere, sorry...)

Reply:
The intent is that the events are automatically enabled for all browsing contexts, and will automatically be enabled for contexts that are newly created.

Reply:
I've updated this and used strings everywhere as you and bwalderman seemed confused by the attempt to use spec-internal objects.

Reply:
Yes. Spec-wise you're going to invoke all events and then filter. Implementations probably won't do that for efficiency reasons, but the models are isomorphic.